core/src/main/java/org/kiwix/kiwixmobile/core/utils/NetworkUtils.kt

```diff
@@ -0,0 +1,111 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2019 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package org.kiwix.kiwixmobile.core.utils
+
+import android.content.Context
+import android.net.ConnectivityManager
+import android.net.NetworkInfo
+import android.os.Build
+import android.util.Log
+import org.kiwix.kiwixmobile.core.R
+import java.lang.Exception
+import java.util.UUID
+
+object NetworkUtils {
+  /**
+   * check availability of any network
+   *
+   * @return true if a network is ready to be used
+   */
+  fun isNetworkAvailable(context: Context): Boolean {
+    val connectivity: ConnectivityManager? = context
```

Comment:
It is a non-nullable type. So we can remove null.

Reply:
hi @gouri-panda  done

Reply:
hi @gouri-panda  done